src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -704,6 +704,16 @@ void TwitchMessageBuilder::appendUsername()
         break;
     }
 
+    auto aliases = *getCSettings().aliasNames.readOnly().get();
+
+    for (auto const alias : aliases)
+    {
+        if (alias.getName().toLower() == this->message().loginName.toLower())
+        {
+            usernameText = alias.getReplace();
+        }
+    }
```

Comment:
Currently if you have 2 aliases for the same user, the last one in the list is used instead of the first one like what seems to be the case for commands. Maybe an early break here?
Might want to just prevent duplicates somehow but that's another thing.